### PR TITLE
Use single Dockerfile and ARG for NODE_VERSION

### PR DESCRIPTION
**Pull request by saberduck**

### CODE REVIEW

File: .cirrus/nodejs.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM eu.gcr.io/release-engineering-ci-prod/base:j11-latest
+
+USER root
+
+ARG NODE_VERSION=v12.22.0
```
Comment (ilia-kebets-sonarsource) [inline .cirrus/nodejs.Dockerfile:5]: Is this correctly overriden by the `docker_arguments` property in Cirrus?

Comment (ilia-kebets-sonarsource) [reply .cirrus/nodejs.Dockerfile:5]: If you click on the check: `Prebuild ....` here https://cirrus-ci.com/task/6073281299087360, you can see that it takes an image from the cache with no version number in the name:

```
Image languages-ci-prod/sonarsource/sonarjs/cirrus/nodejsdockerfile is already cached!
```

Comment (ilia-kebets-sonarsource) [reply .cirrus/nodejs.Dockerfile:5]: I took a look at an older check: https://github.com/SonarSource/SonarJS/pull/3272/checks?check_run_id=7521169541
and the image version was included:

```
Image languages-ci-prod/sonarsource/sonarjs/cirrus/nodejs-12dockerfile is already cached!
```

Comment (saberduck) [reply .cirrus/nodejs.Dockerfile:5]: The older (current master) branch just used version number in the name of the file as a convention to distinguish the files.

I added a small script to print the version of node, you can check here https://cirrus-ci.com/task/6011025748131840?logs=node_version#L1 that it works as expected. If no docker argument is provided, it defaults to v12.

**Review by ilia-kebets-sonarsource — APPROVED** [review]
Thanks @saberduck for implementing this verification! Can you revert this commit and merge? https://github.com/SonarSource/SonarJS/pull/3273/commits/1422c105b4dc79febfad3609fca5bfb6eda89a2b

**Pull request merged**